public/app/plugins/datasource/loki/querybuilder/components/UnwrapParamEditor.tsx

```diff
@@ -0,0 +1,95 @@
+import { isNaN } from 'lodash';
+import React, { useState } from 'react';
+
+import { isValidGoDuration, SelectableValue, toOption } from '@grafana/data';
+import { Select } from '@grafana/ui';
+
+import { getOperationParamId } from '../../../prometheus/querybuilder/shared/operationUtils';
+import { QueryBuilderOperationParamEditorProps } from '../../../prometheus/querybuilder/shared/types';
+import { LokiDatasource } from '../../datasource';
+import { isBytesString } from '../../language_utils';
+import { getLogQueryFromMetricsQuery, isValidQuery } from '../../query_utils';
+import { lokiQueryModeller } from '../LokiQueryModeller';
+import { LokiVisualQuery } from '../types';
+
+export function UnwrapParamEditor({
+  onChange,
+  index,
+  operationIndex,
+  value,
+  query,
+  datasource,
+}: QueryBuilderOperationParamEditorProps) {
+  const [state, setState] = useState<{
+    options?: Array<SelectableValue<string>>;
+    isLoading?: boolean;
+  }>({});
+
+  return (
+    <Select
+      inputId={getOperationParamId(operationIndex, index)}
+      onOpenMenu={async () => {
+        if (datasource instanceof LokiDatasource) {
```

Comment:
this is always `true`, right?

maybe we could add a comment here along the lines of:
`// this check is always true, we do it to make typescript happy` or something similar, to make it easier for the next person reading it understand why the `if`.